pkg/compiler/codegen.go

```diff
@@ -1346,6 +1365,17 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 	return c
 }
 
+func checkTypeAssertWithOK(n ast.Node) error {
+	switch t := n.(type) {
+	case *ast.TypeAssertExpr:
```

Comment:
Do you need a `switch` here?

Reply:
No, it's a remnant from the previously implemented version, fixed.